docs/source/user/aerodyn/driver.rst

```diff
@@ -0,0 +1,497 @@
+    ----- Turbine(1) Blades -----------------------------------------------------------------
+    3               NumBlades(1)          - Number of blades for current rotor (-)
+    0,0,0           BldOrigin_h(1_1)      - Orign of blade 1 wrt. hub origin in hub coordinates (m)
+    0,0,0           BldOrigin_h(1_2)      - Orign of blade 2 wrt. hub origin in hub coordinates (m)
+    0,0,0           BldOrigin_h(1_3)      - Orign of blade 3 wrt. hub origin in hub coordinates (m)
+    0  ,-4,0        BldOrientation_h(1_1) - successive rotations (theta_x, theta_y, theta_z) defining initial orientation of the blade frame from the hub frame such that the "z" is along span, "y" along trailing edge without pitch (azimuth, precone, pitch) (deg)
+    120,-4,0        BldOrientation_h(1_2) - successive rotations (theta_x, theta_y, theta_z) defining initial orientation of the blade frame from the hub frame such that the "z" is along span, "y" along trailing edge without pitch (azimuth, precone, pitch) (deg)
+    240,-4,0        BldOrientation_h(1_3) - successive rotations (theta_x, theta_y, theta_z) defining initial orientation of the blade frame from the hub frame such that the "z" is along span, "y" along trailing edge without pitch (azimuth, precone, pitch) (deg)
+    3.0             BldHubRad_bl(1_1)     - z-offset in blade coordinates of blade 1 where radial input data start (m)
+    3.0             BldHubRad_bl(1_2)     - z-offset in blade coordinates of blade 2 where radial input data start (m)
+    3.0             BldHubRad_bl(1_3)     - z-offset in blade coordinates of blade 3 where radial input data start (m)
+
+
+
+**Turbine motion definition**
+
+The definition of the turbine motion are only used when `AnalysisType=1`, but must always be present in the input file. 
+
+The base motion is given in the same way for basic or advanced geometries.
+The motion of the base may be: fixed (`BaseMotionType=0`), sinusoidal (`BaseMotionType=1`) or arbitrary (`BaseMotionType=2`). 
+The turbine base motion is applied at each time step before applying the initial position and orientation of the turbine base.
+A sinusoidal motion implies that one degree of freedom (`DegreeOfFreedom`) of the turbine base is moving according to a sine function of a given amplitude (`Amplitude`) and frequency (`Frequency`), with zero phase.
+The 6 possible degrees of freedom correspond to translations or rotations of the base frame in global coordinates (g) (e.g. surge, sway, heave, roll, pitch, yaw).
+An arbitrary motion is specified via a CSV file (`BaseMotionFileName`) which contains 19 columns: time, 3 translations (global), three successive rotations (global), 3 translation velocities, 3 rotational velocities (omega, in global), 3 translational accelerations and 3 rotational accelerations (alpha, in global). Example of arbitrary input files are given in :numref:`ad_inputfiles_examples`.
```

Comment:
Add "The displacements/orientations, velocities, and accelerations are not checked internally for consistency."